src/main/java/de/muenchen/allg/itd51/wollmux/core/db/DJDataset.java

```diff
@@ -134,4 +134,7 @@ public void discardLocalOverride(String columnName)
    *           falls dieser Datensatz nicht aus dem LOS kommt.
    */
   public void remove();
+
+  //@Override
+  public String toString();

```

Comment:
Jedes Java-Objekt besitzt eine toString Methode. Ist die Definition im Interface dann notwendig oder wäre ein anderer Name vielleicht besser

Reply:
Vorteil von toString im Interface zu überschreiben ist mmn das es dann nicht zwingend ist die Methode in Klassen implementieren zu müssen die das Interface verwenden.